### PR TITLE
Update the index.md of 'Using_textures_in_WebGL'

### PR DESCRIPTION
**Pull request by Chaos-Enyo**
Fixed incorrect snippets in the index.md of 'Using_textures_in_WebGL', added missing instructions and correct presentations.

### Description

Fixed incorrect snippets in the index.md, added missing instructions and correct presentations. 

### Motivation

Help readers run the demo project correctly.

### CODE REVIEW

File: files/zh-cn/web/api/webgl_api/tutorial/using_textures_in_webgl/index.md [hunk 1]
```diff
@@ -13,29 +13,85 @@ slug: Web/API/WebGL_API/Tutorial/Using_textures_in_WebGL
 
 > **备注：** 值得注意的一点是对纹理的加载同样需要遵循[跨域访问规则](zh-CN/docs/Web/HTTP/Access_control_CORS)；也就是说你只能从允许跨域访问的网址加载你需要的纹理。下面的例子就是支持跨域访问的。
 
-加载纹理的代码如下：
+添加下面两个函数到"webgl-demo.js"中：
```
Comment (fwqaaq) [inline files/zh-cn/web/api/webgl_api/tutorial/using_textures_in_webgl/index.md:16]:
```suggestion
添加下面两个函数到“webgl-demo.js”中：
```

File: files/zh-cn/web/api/webgl_api/tutorial/using_textures_in_webgl/index.md [hunk 2]
```diff
@@ -54,141 +110,250 @@ function handleTextureLoaded(image, texture) {
...
 gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_WRAP_S, gl.CLAMP_TO_EDGE);
 // Prevents t-coordinate wrapping (repeating).
 gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_WRAP_T, gl.CLAMP_TO_EDGE);
+
 ```
 
 现在，当使用以上参数时，兼容 WebGL 的设备就会自动变得可以使用任何分辨率的纹理（当然还要考虑像素上限）。如果不使用上面这些参数的话，任何非 2 的幂纹理使用都会失败然后返回一张纯黑图片。
 
-## 映射纹理到面
+为了加载图像，在我们的`main()`函数中添加对`loadTexture()`函数的调用。这可以在`initBuffers(gl)`调用之后添加。但需要注意的是：浏览器会从加载的图像中按从左上角开始的自上而下顺序复制像素，而WebGL需要按自下而上的顺序——从左下角开始的像素顺序。所以为了防止渲染时图像纹理方向错误，我们还需要调用`pixelStorei()`并将`gl.UNPACK_FLIP_Y_WEBGL`参数设置为`true`，以调整像素顺序，使其翻转成WebGL需要的自下而上顺序。
```
Comment (fwqaaq) [inline files/zh-cn/web/api/webgl_api/tutorial/using_textures_in_webgl/index.md:125]:
```suggestion
为了加载图像，在我们的 `main()` 函数中添加对 `loadTexture()` 函数的调用。这可以在 `initBuffers(gl)` 调用之后添加。但需要注意的是：浏览器会从加载的图像中按从左上角开始的自上而下顺序复制像素，而 WebGL 需要按自下而上的顺序——从左下角开始的像素顺序。所以为了防止渲染时图像纹理方向错误，我们还需要调用 `pixelStorei()` 并将 `gl.UNPACK_FLIP_Y_WEBGL` 参数设置为 `true`，以调整像素顺序，使其翻转成 WebGL 需要的自下而上顺序。
```

根据指南进行检查：

https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md

**Review by yin1999 — COMMENTED** [review]
与 #12838 存在冲突，请作出修改

File: files/zh-cn/web/api/webgl_api/tutorial/using_textures_in_webgl/index.md [hunk 3]
```diff
@@ -152,30 +171,36 @@ function initTextureBuffer(gl) {
...
+  const textureCoordBuffer = initTextureBuffer(gl);
+  // 返回一个对象，包含了初始化好的positionBuffer
+
+  return {
+    position: positionBuffer,
+    // color: colorBuffer,
+    textureCoord: textureCoordBuffer,
+    indices: indexBuffer,
+  };
+}
```
Comment (yin1999) [inline files/zh-cn/web/api/webgl_api/tutorial/using_textures_in_webgl/index.md]: 请注意对照原文

**Review by yin1999 — APPROVED** [review]
请不要在代码块中空出首行和尾行

File: files/zh-cn/web/api/webgl_api/tutorial/using_textures_in_webgl/index.md (hunk 3 above)
Comment (Gerocer) [reply files/zh-cn/web/api/webgl_api/tutorial/using_textures_in_webgl/index.md]: 好的以后会注意

**Pull request merged**